api.py

```diff
@@ -45,22 +45,66 @@ def new_func(*args, **kwargs):
 
 @_deprecated
 def list_object(object_to_list, **kwargs):
+    """
+    This function is deprecated.  Call :py:func:`list_resources` instead.
+    """
     return list_resources(object_to_list.upper().rstrip('S'), **kwargs)
 
 
 def list_resources(resource_type, **kwargs):
+    """
+    List all resources of the given type.
+
+    A convenience method that calls :py:func:`find_resource`.  This function, however, requires the user to specify a resource type.
+
+    Parameters
+    ----------
+    resource_type : string
+        Resource type to fetch.  One of: SUBSTRATE, DATASET, ASSET, LENS_TEMPLATE, ORCHESTRATION.
+    kwargs : key-value
+        host : string
+            The Conduce server's hostname (ex. app.conduce.com)
+        api_key : string
+            The user's API key (UUID).  The user should provide `api_key` or `user` but not both.  If the user provides both `api_key` takes precedent.
+        user : string
+            The user's email address.  Used to look up an API key from the Conduce configuration or, if not found, authenticate via password.  Ignored if `api_key` is provided.
+
+
+    See :py:func:`find_resource` for information on return type.
+    """
     return find_resource(type=resource_type, **kwargs)
 
 
 def list_datasets(**kwargs):
+    """
+    List all accessible datasets.
+
+    A convenience method that calls :py:func:`list_resources` with the ``type`` field set to "DATASET."
+
+    See :py:func:`find_resource` for information on return type.
+    """
     return list_resources('DATASET', **kwargs)
 
 
 def list_substrates(**kwargs):
+    """
+    List all accessible substrates.
```

Comment:
Do we have a definitions/concepts section yet?  We certainly need to explain substrates somewhere

Reply:
Eventually.  It's not part of data ingest.